gst/nnstreamer/tensor_common_pipeline.c

```diff
@@ -95,6 +95,25 @@ gst_tensor_time_sync_set_option_data (tensor_time_sync_data * sync)
   return TRUE;
 }
 
+/**
+ * @brief Internal function to get EOS.
+ */
+static gboolean
+_gst_tensor_time_sync_is_eos (GstCollectPads * collect,
+    tensor_time_sync_data * sync, guint empty)
```

Comment:
This function requires some explanation: what does "empty" mean?

Reply:
I updated description about this function.
The param empty is the number of empty pads. (no buffer in pad)